knx/knxnet/socket.go

```diff
@@ -81,22 +82,52 @@ type RouterSocket struct {
 // ListenRouter creates a new Socket which can be used to exchange KNXnet/IP packets with
 // multiple endpoints.
 func ListenRouter(multicastAddress string) (*RouterSocket, error) {
-	return ListenRouterOnInterface(nil, multicastAddress)
+	return ListenRouterOnInterface(nil, multicastAddress, false)
 }
 
 // ListenRouterOnInterface creates a new Socket which can be used to exchange KNXnet/IP packets with
-// multiple endpoints. The interface is used to send or listen for KNXNet/IP packets. If the
+// multiple endpoints. The interface is used to send or listen for KNXnet/IP packets. If the
 // interface is nil, the system-assigned multicast interface is used.
-func ListenRouterOnInterface(ifi *net.Interface, multicastAddress string) (*RouterSocket, error) {
+func ListenRouterOnInterface(ifi *net.Interface, multicastAddress string, multicastLoopbackEnabled bool) (*RouterSocket, error) {
```

Comment:
Sorry, don't understand your remark, what separate data structure are you referring to?

Reply:
I think it would be better if the arguments are bundled in a data type and then passed to the function instead of passing them individually as arguments. This way the meaning of arguments is better preserved, especially the boolean argument.